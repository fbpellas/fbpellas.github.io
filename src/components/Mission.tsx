import * as React from 'react';
import { BASE_PATH_IMG } from '../constants';

export interface MissionProps { }

const Mission: React.FC<MissionProps> = (_props) => {
  return <div className="block-2">
    <div className="article">
      <h3 className="h3-title">Mission</h3>
      <img
        className="half-img"
        src={`${BASE_PATH_IMG}wall.jpeg`}
        alt="Learn the Art of Speaking American English"
      />
      <p className="margin-top">
        One of the biggest goals for language learners is to learn how to speak with the correct pronunciation of
        their target language. Unfortunately, many English as a second/foreign language (ESL/EFL) curricula do not
        focus on pronunciation, therefore, many teachers lack training in this field. The purpose of this website
        is to help teachers and students understand the basics of pronunciation. By learning pronunciation,
        students can feel more confident in speaking and communicating.
      </p>
    </div>
  </div>
};

export { Mission }