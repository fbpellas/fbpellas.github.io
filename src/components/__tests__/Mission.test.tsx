import React from 'react';
import { render } from '@testing-library/react';
import { Mission } from '../Mission';

test('Mission', () => {
  const { container } = render(<Mission />);
  expect(container).toMatchSnapshot();
});
