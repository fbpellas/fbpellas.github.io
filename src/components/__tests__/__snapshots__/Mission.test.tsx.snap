// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Mission 1`] = `
<div>
  <div
    class="block-2"
  >
    <div
      class="article"
    >
      <h3
        class="h3-title"
      >
        Mission
      </h3>
      <img
        alt="Learn the Art of Speaking American English"
        class="half-img"
        src="https://raw.githubusercontent.com/fbpellas/fbpellas.github.io/main/public/img/wall.jpeg"
      />
      <p
        class="margin-top"
      >
        One of the biggest goals for language learners is to learn how to speak with the correct pronunciation of their target language. Unfortunately, many English as a second/foreign language (ESL/EFL) curricula do not focus on pronunciation, therefore, many teachers lack training in this field. The purpose of this website is to help teachers and students understand the basics of pronunciation. By learning pronunciation, students can feel more confident in speaking and communicating.
      </p>
    </div>
  </div>
</div>
`;
